### PR TITLE
Update installing-ssma-for-db2-client-db2tosql.md

### PR DESCRIPTION
**Pull request by YoheiKawajiri**
The description of the supported z/OS versions appears to be out of date and not updated.

### CODE REVIEW

**Comment by Court72** [comment]
@cpichuka

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/ssma/db2/installing-ssma-for-db2-client-db2tosql.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This topic provides the installation prerequisites and instructions for installi
 
 ## Prerequisites
 
-SSMA is designed to work with DB2 on z/OS version 9.0 and 10.0, DB2 on LUW version 9.8 and 10.1 or later versions, DB2 for i version 7.1 or greater and [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] 2012 or later versions.
+SSMA is designed to work with DB2 on z/OS version 9.0 and 10.0 or later versions, DB2 on LUW version 9.8 and 10.1 or later versions, DB2 for i version 7.1 or greater and [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] 2012 or later versions.
```
Comment (rwestMSFT) [inline docs/ssma/db2/installing-ssma-for-db2-client-db2tosql.md]:
```suggestion
SSMA is designed to work with DB2 on z/OS version 9.0, 10.0, and later versions, DB2 on LUW version 9.8, 10.1, and later versions, DB2 for i version 7.1 and later versions, and [!INCLUDE [sssql11-md](../../includes/sssql11-md.md)] and later versions.
```

**Review by cpichuka — APPROVED** [review]

**Comment by Court72** [comment]
Thanks, @cpichuka. When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

**Comment by cpichuka** [comment]
> Thanks, @cpichuka. When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

#sign-off

**Comment by Court72** [comment]
Due to the upcoming Ignite conference, all merging and publishing is on hold until the morning of 11/15.

This PR is ready for merge on 11/15.

#label: "event-triaged-to-10AM"
#label: "missed-content-complete"

**Pull request merged**